### DISCUSSION TITLE
Issues with plotting recovery results using Gmsh

### DISCUSSION THREAD

**Discussion by Si1verBe11** [wenbinyugroup/.github#19] [VABS Related Discussions] [answered]
To see the results of the recovery analysis, I run `prevabs -i mh104 -h -e` on the ex_airfoil_r example folder and `prevabs -i mh104.xml -d -e -v`.
This resulted in the following situation when Gmsh was run.
![image](https://github.com/user-attachments/assets/b039ff37-6186-48d8-8431-a0516a734ff4)

1. First of all, it looks like the category for postprocessing is incorrect: there's a category despite no displacement results, and the subitems have stress values in them.

Well, I can use the 27 stress strain values anyway, but I have an additional issue with them.

![image](https://github.com/user-attachments/assets/2ac10434-6829-419b-bac2-62d41467c62b)

2. As shown in the figure above, it incorrectly recognizes the maximum value of strain.
Checking the values in the ELE file, the maximum strain value is about 4.82e-8.
I manually put that value in the maximum value position, and the results seem to be plotted correctly.
![image](https://github.com/user-attachments/assets/e5327e78-e4bf-414d-8249-65c0c595eba1)

3. The last question is the most important one.
Issues like these are just inconveniences, but I can use them as it is.
**But, is there no way to plot stress/strain separately for each material in Gmsh?**

**Comment by wenbinyu-PU** [comment, marked as answer]
Why do you need stress/strain for each material? If you want to access the failure, VABS failure analysis might be easier for you to use.

**Reply by Si1verBe11** [reply]
Thanks Prof.Yu, I will utilize that!
